main/app/reactiveLayer/homeostasis/conf/pps.ini

```diff
@@ -0,0 +1,11 @@
+name                                homeostasis
+period                              0.01
+
+[HOMEOSTATIC]
+drives                                      ()
+
+
+# Drives should rather been defined in the allostatic controller
+sample-homeostasisMin          0.25
+sample-homeostasisMax          1.0
+sample-decay                   0.0
```

Comment:
It's not needed. I remove it. It was just a sample drive, but it's just needed in default.ini.